Rubberduck.Deployment.Build/Builders/RegistryEntryBuilder.cs

```diff
@@ -92,11 +92,11 @@ public TypeLibMap ExtractTypeLib(XmlNode node)
                 @="C:\\Github\\Rubberduck\\Rubberduck\\RetailCoder.VBE\\bin\\Debug"
             */
             const string basePath = baseFolder + @"TypeLib\";
-            var libGuid = node.Attributes["Id"].Value;
-            var libDesc = node.Attributes["Description"].Value;
-            var language = node.Attributes["Language"].Value;
-            var major = node.Attributes["MajorVersion"].Value;
-            var minor = node.Attributes["MinorVersion"].Value;
+            var libGuid = node.Attributes["Id"].Value; //mandatory; it should throw if we don't have it.
+            var libDesc = node.Attributes["Description"]?.Value ?? string.Empty;
+            var language = node.Attributes["Language"]?.Value ?? "0";
+            var major = node.Attributes["MajorVersion"]?.Value ?? "1";
```

Comment:
Do we _really_ want this to be 1.0 if it fails? What about something that makes it obvious like version `D.E.A.D`?

Reply:
The issue is that by time we get here, we already have extracted a type library using heat.exe. It is not obliged to have a version specified. According to [the specs](https://msdn.microsoft.com/en-us/library/cc237631.aspx), it looks like the default should be 0.0. So, sorry, no `DE.AD` version for you. You're of course welcome to write your dead type library if you feel strongly about it. 😆